test/jit/test_optimize_for_mobile_preserve_debug_info.py

```diff
@@ -76,10 +76,7 @@ def __init__(
                 conv_transpose2d_weight,
                 conv_transpose2d_bias,
             ):
-                super(
-                    TestPrepackedLinearBeforeInlineAndConv2dOp,
-                    self,
-                ).__init__()
```

Comment:
As this is being parsed by JIT frontend, please keep it as is

Reply:
Done